db/migrate/20170303023308_create_exams.rb

```diff
@@ -0,0 +1,13 @@
+class CreateExams < ActiveRecord::Migration[5.0]
+  def change
+    create_table :exams do |t|
+      t.string :score
+      t.datetime :exam_date
```

Comment:
how else would you keep track of the exams? a student can have many exams at different times.